### ISSUE TITLE
Possible misleading information @pnpx-cli.md

### ISSUE THREAD

**Issue by seahindeniz** [pnpm/pnpm.github.io#118]
Hi, I think this should mention "running packages" at the end of the line instead of installing. This can be misunderstood, I guess.

https://github.com/pnpm/pnpm.github.io/blob/06052774d43af9c2db899607ac70e510ac0c70bf/docs/pnpx-cli.md#L28-L29

**Comment by zkochan** [comment]
But `pnpx` is indeed installing packages.

**Comment by seahindeniz** [comment]
Yeah 😓, I know it stores the package in the store folder. But the reason why I said, is that at the beginning of the page, it mentions that pnpm executes/hotloads the package without installing it and then saying that it installs packages, kind of confusing 🤔
> pnpx (PNPm eXecute) is a command line tool that fetches a package from the registry without installing it as a dependency, hotloads it, and runs whatever default command binary it exposes.

Maybe it can say something like "it installs packages to the store" ?
And since this is page is like one of the first pages, this may lead to another question, "What is the store?"

**Comment by zkochan** [comment]
We can make that page less prominent or remove it. pnpx is confusing and it will be probably deprecated in next major version. There is the `pnpm exec` command instead.
